paddle/fluid/inference/tensorrt/convert/rsqrt_op.cc

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "paddle/fluid/inference/tensorrt/convert/op_converter.h"
+
+namespace paddle {
+namespace framework {
+class Scope;
+
+namespace proto {
+class OpDesc;
+}  // namespace proto
+}  // namespace framework
+}  // namespace paddle
+
+namespace paddle {
+namespace inference {
+namespace tensorrt {
+
+class RSQRTOpConverter : public OpConverter {
+ public:
+  void operator()(const framework::proto::OpDesc& op,
+                  const framework::Scope& scope,
+                  bool test_mode) override {
+    framework::OpDesc op_desc(op, nullptr);
+    auto* X = engine_->GetITensor(op_desc.Input("X")[0]);
+    auto output_name = op_desc.Output("Out")[0];
+
+    nvinfer1::IUnaryLayer* layer = TRT_ENGINE_ADD_LAYER(
+        engine_, Unary, *X, nvinfer1::UnaryOperation::kSQRT);
+    auto* output = layer->getOutput(0);
+    auto ouput_layer = TRT_ENGINE_ADD_LAYER(
+        engine_, Unary, *output, nvinfer1::UnaryOperation::kRECIP);
```

Comment:
rsqrt  cloud be implemented in

https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/inference/tensorrt/convert/unary_op.cc